### PR TITLE
[WIP] Polyfill min/max for SIMD integers

### PR DESCRIPTION
**Pull request by MaxGraey**

### CODE REVIEW

File: src/builtins.ts [hunk 1]
```diff
@@ -340,6 +340,8 @@ export namespace BuiltinSymbols {
   export const i8x16_shr_u = "~lib/builtins/i8x16.shr_u";
   export const i8x16_any_true = "~lib/builtins/i8x16.any_true";
   export const i8x16_all_true = "~lib/builtins/i8x16.all_true";
+  export const i8x16_min = "~lib/builtins/i8x16.min";
```
Comment (dcodeIO) [inline src/builtins.ts]: As these are not actual instructions I'd prefer leaving them out and only do `v128.min`

Comment (MaxGraey) [reply src/builtins.ts]: ok, got it

File: src/builtins.ts [hunk 2]
```diff
@@ -4840,6 +4848,42 @@ export function compileCall(
...
+        case TypeKind.U16:
+        case TypeKind.I32:
+        case TypeKind.U32: {
+        // case TypeKind.I64:
+        // case TypeKind.U64: {
+          let flow = compiler.currentFlow;
+          arg0 = compiler.compileExpression(operands[0], Type.v128, ConversionKind.IMPLICIT, WrapMode.NONE);
+          arg1 = compiler.compileExpression(operands[1], Type.v128, ConversionKind.IMPLICIT, WrapMode.NONE);
+          let tempLocal0 = flow.getTempLocal(Type.v128);
+          let tempLocal1 = flow.getTempLocal(Type.v128);
```
Comment (dcodeIO) [inline src/builtins.ts]: 2nd should be getAndFreeTempLocal

File: src/builtins.ts [hunk 3]
```diff
@@ -4882,6 +4926,42 @@ export function compileCall(
...
```
Comment (dcodeIO) [inline src/builtins.ts]: 2nd should be getAndFreeTempLocal

File: src/builtins.ts (hunk 2 above)
Comment (MaxGraey) [reply src/builtins.ts]: Thanks!

**Comment by MaxGraey** [comment]
This no need any more. See https://github.com/WebAssembly/binaryen/pull/2416

**Pull request closed without merging**